.github/workflows/test.yml

```diff
@@ -32,3 +32,9 @@ jobs:
       - name: "Run build test"
         run: |
           make test
+  pre-commit:
```

Comment:
In other AM related repositories we call the equivalent of this job `lint` or `linting` and we set a human readable `name` attribute for it and its steps. Could we do that here?